Assets/BossRoom/Scripts/Shared/Net/ConnectionManagement/ClientGameNetPortal.cs

```diff
@@ -18,6 +17,8 @@ public class ClientGameNetPortal : MonoBehaviour
         public static ClientGameNetPortal Instance;
         private GameNetPortal m_Portal;
 
+        string m_JoinCode;
```

Comment:
I thought we already had this cached somewhere? (multiple caches of the same value is just asking for desyncs later)